recipes/seqsero2/build.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+$PYTHON setup.py install
```

Comment:
can you move this into the meta.yaml and add a `pip install ....` instead like other python only package?

Reply:
Yes, of course. Thanks!